polyA/converters/cm_to_stockholm.py

```diff
@@ -181,13 +181,17 @@ def print_alignment(chrom_seq, subfam_seq, chrom, subfam, f_out_sto):
     f_out_sto.write("//\n")
 
 
-def print_score_matrix(filename_out_matrix, score_matrix, matrix_name):
+def print_score_matrix(
+    filename_out_matrix, score_matrix, matrix_name, background_freqs
+):
     """
     print score matrix to its own output file with extension ".matrix"
     """
     f_out_matrix = open(filename_out_matrix, "w")
```

Comment:
While we're here, we should change this to use a `with` to open the file, or else close it at the end of the function.